skglm/utils/sparse_ops.py

```diff
@@ -101,3 +101,34 @@ def _sparse_xj_dot(X_data, X_indptr, X_indices, j, other):
     for i in range(X_indptr[j], X_indptr[j+1]):
         res += X_data[i] * other[X_indices[i]]
     return res
+
+@njit(fastmath=True)
+def sparse_subselect_cols(cols, X_data, X_indptr, X_indices):
+    """Summary
+    
+    Parameters
+    ----------
+    cols : TYPE
```

Comment:
I guess you can make this method private (name starting with an underscore) and write a 1 line docstring (no need to describe the parameters)